partiql-eval/src/test/kotlin/org/partiql/eval/internal/PartiQLEngineDefaultTest.kt

```diff
@@ -203,6 +203,20 @@ class PartiQLEngineDefaultTest {
                     )
                 )
             ),
+            SuccessTestCase(
```

Comment:
(Following rebase of partiql-eval) This tests and the `EXCLUDE` tests from L250-315 were inadvertently deleted in #1362.